Content.Shared/_Sunrise/Weapons/Ranged/ConsumableAmmoComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._Sunrise.Weapons.Ranged
```

Comment:
```suggestion
namespace Content.Shared._Sunrise.Weapons.Ranged;
```

Скобочки убрать, код сдвинуть